src/site/markdown/server/manage-webadmin.md

```diff
@@ -1386,6 +1387,92 @@ Response codes:
  - 204: OK
  - 400: Alias structure or member is not valid
 
+## Creating address domain
+
+You can use **webadmin** to define domain mappings.
+
+Given a configured source(from) domain maps to a destination(to) domain. Whenever an email is sent to an address of source domain.
+It will be navigated to the corresponding address at the destination domain.
```

Comment:
Given a configured source (from) domain, mapping to a destination (to) domain, 
When an email is sent to an address belonging to the source domain
Then the domain part of this address is overwritten, the destination domain is then used.